.github/workflows/tests.yml

```diff
@@ -148,6 +148,7 @@ jobs:
       matrix:
         target:
           - "python_coverage"
+          - "python_unit_coverage"
         python-version:
           - "3.8"
           - "3.9"
```

Comment:
I think it should be sufficient to run the analysis of the unit test coverage just for one python version. We could replace the existing `python_unit` target in the `include` section.

Reply:
Changed.